assets/common/scripts/game_manager.gd

```diff
@@ -15,12 +15,18 @@ const grid_props_dt: Datatable = preload("res://assets/content/grid_props_dt.tre
 
 const crop_scn = preload("res://assets/crops/scenes/crop.tscn")
 
+var scanner_prop: String:
+	set(prop):
+		scanner_prop = prop
+		scanner_prop_updated.emit(prop)
+
 var game_world: GameWorld:
 	set(world):
 		Utils.log_info("Initialisation", "Game world registered")
 		game_world = world
 
 func _ready():
+	ZoneLayouts.load_file()
 	Savegame.load_file()
```

Comment:
Is this correct order of dependency?

Reply:
Yep, because the savegames loading may potentially need to access some of the static zone layout info